### PR TITLE
Improve wording of delete actions (EXPOSUREAPP-10407)

### LINKED ISSUE #4323: "Delete All Permanently" for single certificate in Recycle Bin
## Avoid duplicates
* [X] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Bug is specific for Android only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Bug is not already reported in another issue

## Technical details

- Android version: 11
- First reported on App version: 2.13.1-RC1
- Confirmed still present on App version: 2.13.2

## Describe the bug

The options for a certificate in the Recycle Bin are:

- "Restore"
- "Delete All Permanently"

For reference see screenshot https://www.coronawarn.app/assets/screenshots/2-13/en/android/RecyclerBinOverviewFragment_full_2.png.

The word "All" does not make sense in this context because selecting this option deletes only one certificate, not all certificates.

To remove all certificates there is a separate option available at the top-level of the Recycle Bin which is labelled "Remove All".

## Steps to reproduce the issue

1. Scan in two certificates
2. Remove both certificates
3. Go to the Recycle bin
4. For one certificate select "Delete All Permanently"
5. Note that only one certificate has been deleted from the recycle bin, not all certificates

## Expected behavior

The menu text for a single certificate should only apply to the one certificate.

## Possible Fix

Change the text in string `recycle_bin_remove_permanently`
from "Delete All Permanently" to "Delete Permanently"

https://github.com/corona-warn-app/cwa-app-android/blob/5ea5c9c329e3ae31598bd476716e1348e6d44169/Corona-Warn-App/src/main/res/values/recycler_bin_strings.xml#L35-L36

---
Internal Tracking ID: [EXPOSUREAPP-10407](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10407)

### PR DESCRIPTION
**Pull request by marcauberer**
Re-addresses [EXPOSUREAPP-10407](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10407) and fixes #4323

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
     <!-- XTXT: Recycle Bin - restore -->
     <string name="recycle_bin_restore">"Wiederherstellen"</string>
     <!-- XTXT: Recycle Bin - remove selected item permanently -->
-    <string name="recycle_bin_remove_permanently">"Endgültig löschen"</string>
+    <string name="recycle_bin_remove_permanently">"Alle endgültig löschen"</string>
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: This is part of the context menu for the selected item in recycle bin. Word "Alle" or "All" doesn't make sense here.

**Review by jurajkusnier — CHANGES_REQUESTED** [review]
I think the issue has been already fixed here https://github.com/corona-warn-app/cwa-app-android/pull/4251 
German translations in `release/2.14.x` branch are according to the Figma design

File: Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml (hunk 1 above)
Comment (mtwalli) [reply Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: @CV113 @marcauberer as Juraj mentioned this is for deleting an item from the list , does it make sense to say delete ALL ?

Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: @KathrinLuecke FYI. What do you think?

Comment (KathrinLuecke) [reply Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: No, it does not make sense. But I corrected it with https://github.com/corona-warn-app/cwa-app-android/pull/4324. Why does it come up again?

Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: The screenshot in the ticket is from the English translation. And indeed English translation is wrong, but this should be fixed by translation delivery for 2.14

Comment (marcauberer) [reply Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml:37]: @KathrinLuecke see the Jira ticket

**Comment by mtwalli** [comment]
The issue here is already fixed in 2.14 using right terms also
- https://github.com/corona-warn-app/cwa-app-android/blob/e3250b766dc6b12dabd9d4d817da2a875a407ab7/Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml#L14
- https://github.com/corona-warn-app/cwa-app-android/blob/release/2.14.x/Corona-Warn-App/src/main/res/values-de/recycler_bin_strings.xml#L37

**Pull request closed without merging**